src/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions/SchedulingMode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions
+{
+    public enum SchedulingMode
+    {
+        Default,
```

Comment:
q: Why do we need default? Can't we do with the other two? Seems like default is ambiguous.